searchbase/lib/src/searchcontroller.dart

```diff
@@ -1040,6 +1047,8 @@ class SearchController extends Base {
       return Future.value(this.recentSearches);
     } catch (e) {
       return Future.error(e);
+    } finally {
```

Comment:
Closing the `httpClient` here has the same-effect of re-instantiating it per request.